index.js

```diff
@@ -1,64 +1,92 @@
+                            'provide a song to search for!'
+                };
+            }
+
+            // the response body as JSON
+            const data = await get(base + args).then(res => JSON.parse(res.body));
+
+            // response had no ``data`` key, which means the query returned nothing
+            if (!data.data) {
+                return {
+                    send: false,
+                    result: "I couldn't find that song!"
+                };
+            }
+
+            let lyrics;
+
+            // if the lyrics are more than 2000 characters, the embed's description
+            // will simply be a hyperlink to the lyrics
+            if (data.data[0].lyrics.length > 2000) {
+                lyrics = `[Click Here](${data.data[0].url})`;
+            } else {
+                // not sure why this regex is here but yes
+                lyrics = data.data[0].lyrics.replace(/(?:\\[rn])+/g, '');
             }
-            data = await get(`https://lyrics-api.powercord.dev/lyrics?input=${args}`).then(res => JSON.parse(res.body));
-          }
-          if (!data.data[0].lyrics) {
+
+            // builds the embed with our data
+            const embed = {
+                type: 'rich',
+                title: `${data.data[0].artist} - ${data.data[0].name}`,
+                color: Math.floor(Math.random() * 16777215),  // random colour
+                description: lyrics,
+                footer: {
+                    text: `Lyrics provided by KSoft.Si | © ${data.data[0].artist} ${data.data[0].album_year}`
+                }
+            };
+
             return {
-              send: false,
-              result: 'I couldn\'t find that song!'
+                send: false,
+                result: embed
             };
-          }
-          data = `${data.data[0].artist} - ${data.data[0].name}\n\n${data.data[0].lyrics}\n\nLyrics provided by KSoft.Si | © ${data.data[0].artist} ${data.data[0].album_year}`;
-          const value = data.replace(/(?:\\[rn])+/g, '');
-          // ...then send it
-          if (value.length > 2000) { // Post link in chat if the lyrics are over 2000 characters
-            return messages.sendMessage(
-              channels.getChannelId(),
-              { content: data.data[0].url }
-            );
-          }
-          messages.sendMessage(
-            channels.getChannelId(),
-            { content: `\`\`\`${value}\`\`\`` }
-          );
+
         } catch (e) {
-          console.log(e);
-          return {
-            send: false,
-            result: 'Yikes, I couldn\'t find that song due to an error. Please check the Discord developer tools console! (Ctrl + Shift + I)'
-          };
+            console.error('Error with lyrics plugin:', e);
+            return {
+                send: false,
+                result: 'Yikes, something broke. Please check the Dev Console for info.'
+            };
         }
-      });
-  }
+    }
 };
+
+module.exports = Lyrics;
```

Comment:
Please revert this to the way it was before

Reply:
Sure thing